TeamSnapSDK/SDK/DataTypes/TSDKUser.h

```diff
@@ -31,6 +31,7 @@ typedef NS_CLOSED_ENUM(NSInteger,TSDKHighestRoleType) {
 @property (nonatomic, strong, nullable) NSString * addressCountry; //Example:
 @property (nonatomic, strong, nullable) NSString * highestRole; //Example: league_owner
 @property (nonatomic, strong, nullable) NSString * personUuid; //Example: **NULL**
+@property (nonatomic, strong, nullable) NSString * ppid; //Example: 37fed216826d68e64f1f0b1d299216776dde9b31d62ed8a024ca1aad8f68687f
```

Comment:
MINOR
Any reason this can't be ppId or ppID ?Unsure if this determined by the JSON coming back or whatever conventions we have currently.